### DISCUSSION TITLE
Totally fresh install with no prior package list

### DISCUSSION THREAD

**Discussion by b9Joker108** [conda-forge/miniforge#692] [General]
I had problems with miniconda3 that I couldn't fix. So, transitioned to miniforge3. I now have problems in miniforge3 I can't fix.  

I just want to start totally fresh, now I know much more about managing the environment. I want to initiate a fresh install withour any prior package manifest, but cannot figure out how to clear the package list from automatically installing when doing a reinstall.

Any assistance greatly appreciated.

**Comment by hmaarrfk** [comment]
Follow https://github.com/conda-forge/miniforge?tab=readme-ov-file#uninstallation
and delete the `.condrc` in your home directory
